CSharp/Library/Microsoft.Bot.Builder/Dialogs/PromptDialog.cs

```diff
@@ -434,16 +434,28 @@ public sealed class PromptConfirm : Prompt<bool, string>
             /// Index of no descriptions.
             /// </summary>
             public const int No = 1;
-
+            
             /// <summary>
             /// The yes, no options for confirmation prompt
             /// </summary>
-            public static string[] Options { set; get; } = { Resources.MatchYes.SplitList().First(), Resources.MatchNo.SplitList().First() };
```

Comment:
We could leave the setter by having the a private variable that hold the value set and then in the getter have some logic that returns the variable or the defaults

```
public static string[] Options
{
    get
    {
        if (this.options == null)
        {
            return new string[] { Resources.MatchYes.SplitList().First(), Resources.MatchNo.SplitList().First() };
        }

        return this.options;
    }
    set
    {
        this.options = value;
    }
}
```

Though it seems better to just make these properties private and non-static properties and have parameters in the constructor that overrides them when provided. If they are not provided we will default to the values in the Resources. Though this will imply some change in the static Confirm method of the PromptDialog class since it's using th[e static Options property](https://github.com/southworkscom/BotBuilder/blob/master/CSharp/Library/Microsoft.Bot.Builder/Dialogs/PromptDialog.cs#L298).